assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -432,6 +432,9 @@
                  	</a>
 				</p>
 				<div id="allowResubmission_shownInner" style="display:none">
+				#if ($allowReviewService && $assignment.getContent().AllowReviewService)
```

Comment:
Isn't this only specific to TII, eg Vericite may still accept submissions if late.